### PR TITLE
Enum coercion fix

### PR DESCRIPTION
**Pull request by xperiandri**

### CODE REVIEW

**Comment by xperiandri** [comment]
@johnberzy-bazinga, it is strange that I have the same error locally with SDK 2.1.607

**Comment by xperiandri** [comment]
@ivelten, do you review pull requests?

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version" : "2.1.401"
+    "version" : "2.1.607"
```
Comment (ivelten) [inline global.json]: Not sure about this, but sometimes in my previous Pull Requests, I had this same error build when I tried to upgrade the .NET SDK version. Appveyor and TravisCI sometimes don't support specific versions. Try changing back the version to see if it builds successfully.

Comment (xperiandri) [reply global.json]: Succeeded

**Comment by xperiandri** [comment]
So what about a new package version with this fix?

**Comment by johnberzy-bazinga** [comment]
@xperiandri Thanks!

**Comment by xperiandri** [comment]
@ivelten, @johnberzy-bazinga do you have any remarks?

**Pull request merged**